src/redux/actions/dragonsActions.js

```diff
@@ -0,0 +1,22 @@
+export const DISPLAY_DRAGONS = 'space-travelers-hub/dragons/DISPLAY_DRAGONS';
+export const TOGGLE_DRAGON_RESERVATION = 'space-travelers-hub/dragons/TOGGLE_DRAGON_RESERVATION';
+
+const basicURL = 'https://api.spacexdata.com/v3/dragons';
+
+export const displayFetchedDragons = (dragons) => ({
+  type: DISPLAY_DRAGONS,
+  dragons,
+});
+
+export const toggleDragonReservation = (id) => ({
+  type: TOGGLE_DRAGON_RESERVATION,
+  id,
+});
+
+export const fetchDragons = () => (dispatch) => {
+  fetch(basicURL)
+    .then((resolve) => resolve.json())
+    .then((data) => {
+      dispatch(displayFetchedDragons(data));
```

Comment:
- [ ] Upon fetching the data kindly ensure that only the selected data (`id`, `name`, `type`, `flickr_images`) is dispatched to the store. Right now you have unnecessary data in your store. Kindly filter for only the needed data.

  Data to dispatch | Your dispatched data
  :---:|:---:
  ![reem-lab-dragons-data](https://user-images.githubusercontent.com/36542986/156485901-bd653c28-5b81-457d-8c57-bd6c396907be.jpg) | ![reem-lab-dragons](https://user-images.githubusercontent.com/36542986/156485896-6415f83b-cf85-4099-89f7-f02ce3062fbf.jpg)